scripts/linters/pre_commit_linter.py

```diff
@@ -710,13 +710,13 @@ def main(args=None):
         _print_summary_of_error_messages(lint_messages)
         linter_utils.print_failure_message('\n'.join([
             '---------------------------',
-            'Checks Not Passed.',
+            'Linter checks Failed.',
             '---------------------------']))
         sys.exit(1)
     else:
         linter_utils.print_success_message('\n'.join([
             '---------------------------',
-            'All Checks Passed.',
+            'All linter checks Passed.',
```

Comment:
The capitalization here is inconsistent. Please use either all sentence case (e.g. "Linter checks failed.") or all title case (e.g. "Linter Checks Failed.")

Reply:
Fixed